services/streammanager/kafka/client/producer.go

```diff
@@ -12,7 +12,9 @@ import (
 type ProducerConfig struct {
 	ClientID string
 	WriteTimeout,
-	ReadTimeout time.Duration
+	ReadTimeout,
+	DefaultOpTimeout time.Duration
```

Comment:
major: we need to make sure this is never 0, i.e. set a default timeout of 60 seconds in `defaults()`
minor: `ProduceTimeout` sounds more straightforward to me.